OptimizelySDKCore/OptimizelySDKCore/OPTLYDecisionService.m

```diff
@@ -164,7 +164,7 @@ - (NSString *)getBucketingId:(NSString *)userId
     // If the bucketing ID key is defined in attributes, then use that
     // in place of the userID for the murmur hash key
     if (![OPTLYDecisionService isEmptyString:attributes[OptimizelyBucketId]]) {
```

Comment:
how about if we renamed that to getValidString where it returns the string as a string if it is valid or null?  Then, you wouldn't have that weird cast (that gives a warning in xcode).  

String validBucketingId = [OPTLYDecisionService isEmptyString:attributes[OptimizelyBucketId]];
if (validBucketingId != null)  {
  bucketingId = validBucketingId

Reply:
Please review.